### PR TITLE
new changes to the project

### PR DESCRIPTION
**Pull request by abukhshba**

### CODE REVIEW

**Comment by abukhshba** [comment]
hello

File: file.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+data
+mahmoud
```
Comment (khaledfathi) [inline file.txt:2]: require changes

File: new.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+hghghjhdfhdjghdf
```
Comment (khaledfathi) [inline new.txt:1]: reemove this

**Review by khaledfathi — CHANGES_REQUESTED** [review]

File: new.txt (hunk 2 above)
Comment (abukhshba) [reply new.txt:1]: no

**Comment by abukhshba** [comment]
thank you

**Pull request merged**